### PR TITLE
Add all block types

### PR DESCRIPTION
**Pull request by thewilkybarkid**

### CODE REVIEW

File: src/Model/Block/Listing.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace eLife\ApiSdk\Model\Block;
+
+use eLife\ApiSdk\Model\Block;
+
+final class Listing implements Block
```
Comment (thewilkybarkid) [inline src/Model/Block/Listing.php:7]: `List` is a reserved word.

**Comment by thewilkybarkid** [comment]
Ping @giorgiosironi, Proofreader should ignore real  —able words.

**Comment by giorgiosironi** [comment]
We can either add a whitelist (which gets boring) or some logic for the length of the word (must be at least 6-7 letters to qualify for the check); but they're not ultimate solutions. The exception is managed by adding to the class the annotation `@SuppressWarnings(ForbiddenAbleSuffix)`, where the arguments to `SuppressWarnings` can be found at https://github.com/elifesciences/proofreader-php/blob/master/ruleset.xml.dist#L39

**Comment by thewilkybarkid** [comment]
@giorgiosironi Something's a bit wrong with the Alfred build

**Comment by giorgiosironi** [comment]
PHPUnit effectively has wrong permissions:

```
elife@ip-10-0-2-117:/var/lib/jenkins-libraries-runner/workspace/pull-requests-libraries/api-sdk-php/PR-10$ ls -l vendor/phpunit/phpunit/phpunit
-rw-r--r-- 1 elife elife 1150 Feb  4  2016 vendor/phpunit/phpunit/phpunit
```

I suspect there is a too-aggressive chmod in the server's formula.

**Comment by giorgiosironi** [comment]
https://github.com/elifesciences/elife-libraries-formula/commit/b0e24005ffdabe2ff447b79d2a4bafa6c85f981a

**Pull request merged**